pkg/bugzilla/client.go

```diff
@@ -0,0 +1,80 @@
+// Package bugzilla implements a basic bugzilla jsonrpc api wrapper with a single query
```

Comment:
if you're going to have doc comments on packages, convention is to only put them in one file.  Convention is to have a file called `doc.go` that looks like

```go

// package foo does bar
package foo
```

and skip the doc comment elsewhere.